src/java.desktop/share/native/libfontmanager/freetypeScaler.c

```diff
@@ -298,6 +299,19 @@ static void setInterpreterVersion(FT_Library library) {
 #if defined(_WIN32) || defined(__APPLE__)
     FT_Property_Set(library, module, property, (void*)(&version));
 #else
+
+    if (JVM_IsStaticallyLinked()) {
```

Comment:
These kinds of checks on behalf of a statically linked build are quite intrusive.
Perhaps there is no alternative, but still.
SFAIK, at this time, no one is under any obligation to test this works, or to know how not to break it.
So I will approve with those caveats.

Reply:
Thanks for the review, @prrace!